ui-ngx/src/app/modules/home/components/widget/lib/home-page/getting-started-widget.component.scss

```diff
@@ -118,7 +118,7 @@
           padding: 0;
           pre[class*=language-] {
             margin: 0;
-            padding: 9px 38px 9px 16px;
+            padding: 10px 38px 10px 16px;
```

Comment:
Previous style correct work in screen width 1800px

Reply:
its still works fine with 1800px 
![image](https://github.com/user-attachments/assets/c13e1bcb-b3b6-48e0-b251-49c7bfcf1c67)